Patch is project specific, and not a candidate for drupal contribution.
This patch checks for null in redirects that could potenitally
be imported from VSF team one time csv import list.
diff --git a/modules/headless_ui/src/Redirect.php b/modules/headless_ui/src/Redirect.php
index 5b0301b..6148be4 100644
--- a/modules/headless_ui/src/Redirect.php
+++ b/modules/headless_ui/src/Redirect.php
@@ -11,10 +11,11 @@ class Redirect {
   public static function entityForm(array &$form, FormStateInterface $form_state) {
     /** @var \Drupal\Core\Entity\EntityFormInterface $form_object */
     $form_object = $form_state->getFormObject();
+    $form_id = is_null($form_object->getBaseFormId()) ? $form_object->getFormId() : $form_object->getBaseFormId();

     $redirect = [
       static::class,
-      Inflector::camelize($form_object->getBaseFormId()),
+      Inflector::camelize($form_id),
     ];

     if (is_callable($redirect)) {
